src/utils/config.ts

```diff
@@ -386,3 +386,11 @@ export const ALLOWED_SEARCH_LANGS = {
   fr: 'french',
   it: 'italian',
 };
+
+// Geolocation API Key
+if (!process.env.GEOLOCATION_API_KEY) {
```

Comment:
Add in the env description in the Readme ? Also once stable it should be added to the docs. Will this be required when running the backend ? Can we provide any value if we do not plan to use that feature ?